mockintosh/handlers.py

```diff
@@ -660,10 +665,19 @@ def __init__(self):
         self.path = None
         self.headers = {}
         self.queryString = {}
-        self.body = None
+        self._json = NotParsedJSON()
         self.files = {}
         self.formData = {}
 
+    @property
+    def json(self):
+        if isinstance(self._json, NotParsedJSON):
+            try:
+                self._json = json.loads(self.body)
+            except json.JSONDecodeError:
```

Comment:
How will user know about the problem? No logging, no messages passed anywhere... That will be a confusing product

Reply:
Added logging with https://github.com/up9inc/mockintosh/commit/d7dcadaf71c26aa74e30782b060ad355980150b2